granian/server.py

```diff
@@ -95,6 +96,11 @@ def __init__(
         respawn_failed_workers: bool = False,
         respawn_interval: float = 3.5,
         reload: bool = False,
+        reload_paths: Sequence[pathlib.Path] = (pathlib.Path.cwd(),),
+        reload_ignore_dirs: Optional[Sequence[str]] = (),
+        reload_ignore_entity_patterns: Optional[Sequence[str]] = (),
+        reload_ignore_paths: Optional[Sequence[pathlib.Path]] = (),
```

Comment:
I would rather make all of these parameters `Optional[...] = None` and set the default later on the `self.var = ` step.

Reply:
Done!